src/DataCollector/ViewCollector.php

```diff
@@ -68,7 +68,7 @@ public function addView(View $view)
         // Prevent duplicates
         $hash = $type . $path . $name . $this->collect_data ? implode(array_keys($view->getData())) : '';
 
-        if (class_exists('\Inertia\Inertia') && isset($data['page'], $data['component'])) {
+        if (class_exists('\Inertia\Inertia') && isset($data['page'], $data['page']['component'])) {
             $data = $data['page'];
             $name = $data['component'];
```

Comment:
Yes
https://www.php.net/manual/es/function.isset.php
**Demo:** https://paiza.io/projects/CXDnryRn_uxM_LS4SUhwnQ

Reply:
Yeah okay, that's what I meant :P I'm just not sure how the Inertia config looks like. Don't have a demo project ready, but will try.

Reply:
I have the same problem, i just follow user indications
https://github.com/barryvdh/laravel-debugbar/issues/1389#issuecomment-1764059972
![image](https://github.com/barryvdh/laravel-debugbar/assets/4933954/26123d44-481c-4024-96f9-5fd0a3d968d9)
You could add the other keys(`prop/version`) for more security, although I think it would not be necessary